Sources/SwiftUIKit/Views/Table.swift

```diff
@@ -66,8 +66,13 @@ extension Table: UITableViewDataSource {
         let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
         
         configureCell?(cell)
-        cell.contentView.embed {
-            self.data[indexPath.row]
+        cell.contentView.clear()
+            .embed { self.data[indexPath.row] }
+        
+        // Start LoadingViews
+        if let view = cell.contentView.allSubviews.first(where: { $0 is LoadingView }),
```

Comment:
Going to debate this for a bit before merging 🤔

Reply:
Alright I have decided to leave as-is for Table.. Table should be used for static content and custom views.

DynamicTable should be used for dynamic content